resolversrv/models.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from flask import current_app
+from sqlalchemy import Integer, String, Column, and_
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm.exc import NoResultFound
+
+
+Base = declarative_base()
+
+class DataLinks(Base):
+    __tablename__ = 'datalinks'
+    __table_args__ = ({"schema": "nonbib"})
+    bibcode = Column(String, primary_key=True)
+    link_type = Column(String, primary_key=True)
+    link_sub_type = Column(String, primary_key=True)
+    url = Column(ARRAY(String))
+    title = Column(ARRAY(String))
+    item_count = Column(Integer)
+
+    def __init__(self, bibcode, link_type, link_sub_type, url, title, item_count):
+        """
+
+        :param bibcode:
+        :param link_type:
+        :param link_sub_type:
+        :param url:
+        :param title:
+        :param item_count:
+        :return:
+        """
+        self.bibcode = bibcode
+        self.link_type = link_type
+        self.link_sub_type = link_sub_type
+        self.url = url
+        self.title = title
+        self.item_count = item_count
+
+    def toJSON(self):
+        """
+
+        :return: values formatted as python dict, if no values found returns empty structure, not None
+        """
+        return {
+            'bibcode': self.bibcode,
+            'link_type': self.link_type or '',
+            'link_sub_type': self.link_sub_type or '',
+            'url' : self.url or [''],
+            'title': self.title or [''],
+            'itemCount': self.item_count or 0,
+        }
+
+
+def get_records(bibcode, link_type=None, link_sub_type=None):
```

Comment:
please move to `app.py` or some utils; `models.py` should only contain the object/table definitions

Reply:
moved to utils